nova_vm/src/ecmascript/abstract_operations/type_conversion.rs

```diff
@@ -427,6 +427,18 @@ pub(crate) fn to_big_int(agent: &mut Agent, argument: Value) -> JsResult<BigInt>
     todo!()
 }
 
+/// ### [7.1.14 StringToBigInt ( str )](https://tc39.es/ecma262/#sec-stringtobigint)
+pub(crate) fn string_to_big_int(agent: &mut Agent, argument: Value) -> JsResult<Value> {
```

Comment:
**issue:** Spec text says that this operation returns a `BigInt` or undefined. We'll convert that to the most convenient form we can which is then `Option<BigInt>`.